be/src/storage/rowset/storage_page_decoder.cpp

```diff
@@ -59,6 +66,7 @@ class BitShuffleDataDecoder : public DataDecoder {
                                         compressed_body.size, bytes));
         }
 
+        auto null_size = get_null_size(footer);
```

Comment:
For dictionary_page, what's the result of of get_null_size(footer) ?

Reply:
Why not directly fetch from data_footer.nullmap_size();?

Reply:
For dictionary page, there is no nullmap size field defined, check to segment.proto.DictPageFooterPB
BitShuffleDataDecoder could decode data page or dictionary page, for dictionary page we just set the nullmap_size = 0

Reply:
Why not use __isset to check?